pkg/cmd/roachtest/testdata/regression.diffs

```diff
@@ -60028,21 +60051,25 @@ diff -U3 --label=/mnt/data1/postgres/src/test/regress/expected/create_procedure.
 - 1
 -(1 row)
 -
++ERROR:  unknown signature: public.ptest9(unknown)
```

Comment:
I wonder is this a known issue that we can't just pass NULL to a procedure?

Reply:
I think this broke because we currently have proper handling of OUT parameters only for UDFs and not for stored procedures. In other words, the failure here is expected at the moment (I'll be working on stored procs support this week).

Reply:
Sounds good, thank you!